libs/ledger-live-common/src/families/cosmos/crypto/Cosmos.ts

```diff
@@ -0,0 +1,159 @@
+import { makeLRUCache } from "../../../cache";
+import network from "../../../network";
+import { CosmosRewardsState } from "../types";
+import cosmosBase from "./cosmosBase";
+import { parseUatomStrAsAtomNumber } from "../logic";
+
+class Cosmos extends cosmosBase {
```

Comment:
```suggestion
class Cosmos extends CosmosBase {
```
Let's keep it PascalCase for classes

Reply:
fixed